data-platform/autonomous-database/autonomous-serverless/README.md

```diff
@@ -25,6 +28,68 @@ Oracle Cloud provides a set of data management services built on self-driving Or
  
 - [Manage and Monitor Autonomous Database](https://apexapps.oracle.com/pls/apex/r/dbpm/livelabs/view-workshop?wid=553&clear=RR,180&session=115113440340155)
     - Complete the course about how to administrate an Autonomous Database.
+
+# Tip of the Day
+ 
+- [Tip 1](https://www.linkedin.com/posts/javier-de-la-torre-medina_oracle-autonomousdatabase-tipoftheday-activity-7104394940436164609-LSa-?utm_source=share&utm_medium=member_desktop)
```

Comment:
This applies to all tips:

- Did you get a legal approval to use these non-Oracle resources?
- Remove all "utm_" query parameters